Quickstart.txt

```diff
@@ -66,21 +65,26 @@ button.
 Use the cursor keys to move the camera in external and sidereal view. In
 sidereal view, use ',' (comma) and '.' (period) to roll the camera in place.
 
-To jump to another system, use the map (F2) and click on another system. Systems
-with ports are labelled in blue. You need enough Hydrogen to get there, and it
-must be close enough for your ship. F7 will start the hyperdrive.
+To jump to another system, use the map (F2) and click on another system. The
+map will provide you with info about the system, such as population, faction
```

Comment:
You also need to add it to your route. That caused confusion a couple of times if I remember correctly.

Reply:
Ah, true. Good find.

Reply:
I changed it, you may check if it is good.

Reply:
Sounds good.